### PR TITLE
Update the N2 docs to reflect all the recent changes to the N2 diagram

### LINKED ISSUE #1319: Docs on N2 are slightly out of date.
### Summary of Issue

The docs reference some of the older icons now. Need to be updated. 

### Issue Type

- [ ] Bug
- [x ] Enhancement  
     

### Description

Check over all docs with N2, and make sure they match with the new skin

### Example
See the road-sign icon that is referenced in the doc on this page: 
http://openmdao.org/twodocs/versions/latest/features/model_visualization/n2_details.html#model-hierarchy

Screen Shot 2020-04-10 at 5 13 31 PM

### PR DESCRIPTION
**Pull request by hschilling**
### Summary

There have been a lot of changes in the N2 lately including the new toolbar. The N2 docs were not updated to reflect those changes. 

### Related Issues

- Resolves #1319

### Backwards incompatibilities

None

### New Dependencies

None

### CODE REVIEW

File: openmdao/docs/features/model_visualization/n2_details.rst [hunk 1]
```diff
@@ -79,183 +84,316 @@ In the upper triangular, the marked cells represent downstream connections. The
...
+Here are explanations of the sections in the legend:
+
+    * The System & Variables section shows the colors for the different items in the model hierarchy.
+
+    * The N2 Symbols section shows the icons used to indicate the type of connection in the connection matrix.
+      The shape of the
+      icon in the matrix shows whether the connection is between scalars, vectors, or groups. The color of the icon is based
+      on the color associated with the variable of that type as shown in the System & Variables part of the legend.
+
+    * Finally, there is a section describing the colors for the items in the solver hierarchy on the right of the
```
Comment (Kenneth-T-Moore) [inline openmdao/docs/features/model_visualization/n2_details.rst:348]: make parallel with the other bullets, e.g.: "The Solvers section shows the colors used for the different  solvers on the right side of the N2.  Either the Linear or Nonlinear solvers are shown depending on the state of the Toggle Solver Names button."

File: openmdao/docs/features/model_visualization/n2_details.rst [hunk 2]
```diff
@@ -79,183 +84,316 @@ In the upper triangular, the marked cells represent downstream connections. The
...
+   | |model_height|        | Model Height                    | Set the height of the diagram in pixels                                                            |
+   +-----------------------+---------------------------------+----------------------------------------------------------------------------------------------------+
+   | |save_svg|            | Save SVG                        | Save the current view of the diagram to an SVG file                                                |
+   +-----------------------+---------------------------------+----------------------------------------------------------------------------------------------------+
+   | |show_node_info|      | Show / Hide Node Info           | Turning this on while result in information about the various nodes to be displayed while hovering |
+   +-----------------------+---------------------------------+----------------------------------------------------------------------------------------------------+
+   | |help|                | Help                            | Brings up the help window                                                                          |
```
Comment (Kenneth-T-Moore) [inline openmdao/docs/features/model_visualization/n2_details.rst:324]: match tense: brings -> bring

File: openmdao/docs/features/model_visualization/n2_details.rst [hunk 3]
```diff
@@ -79,183 +84,316 @@ In the upper triangular, the marked cells represent downstream connections. The
...
+   | |collapse_depth|      | Set collapse depth              | Set how many levels are shown in the System hierarchy on the left                                  |
+   +-----------------------+---------------------------------+----------------------------------------------------------------------------------------------------+
+   | |model_height|        | Model Height                    | Set the height of the diagram in pixels                                                            |
+   +-----------------------+---------------------------------+----------------------------------------------------------------------------------------------------+
+   | |save_svg|            | Save SVG                        | Save the current view of the diagram to an SVG file                                                |
+   +-----------------------+---------------------------------+----------------------------------------------------------------------------------------------------+
+   | |show_node_info|      | Show / Hide Node Info           | Turning this on while result in information about the various nodes to be displayed while hovering |
```
Comment (Kenneth-T-Moore) [inline openmdao/docs/features/model_visualization/n2_details.rst:322]: maybe something like "Toggle info box that is displayed on hover" ?

File: openmdao/docs/features/model_visualization/n2_details.rst [hunk 4]
```diff
@@ -79,183 +84,316 @@ In the upper triangular, the marked cells represent downstream connections. The
...
+   +=====================+==================================+========================================================+
+   | |collapse_view|     | Collapse variables in view only  | Collapse only the variables in the current view        |
+   +---------------------+----------------------------------+--------------------------------------------------------+
+   | |expand_view|       | Expand variables in view only    | Expand only the variables in the current view          |
+   +---------------------+----------------------------------+--------------------------------------------------------+
+   | |collapse_all|      | Collapse all variables           | Collapse all the variables in the entire model         |
+   +---------------------+----------------------------------+--------------------------------------------------------+
+   | |expand_all|        | Expand all variables             | Expand all the variables in the entire model           |
+   +---------------------+----------------------------------+--------------------------------------------------------+
+   | |collapse_depth|    | Collapse depth                   | Sets the number of hierarchy levels shown in the view  |
```
Comment (Kenneth-T-Moore) [inline openmdao/docs/features/model_visualization/n2_details.rst:224]: Sets -> Set

Comment (hschilling) [reply openmdao/docs/features/model_visualization/n2_details.rst]: thanks. Fixed them all

**Pull request merged**